api.swagger.yaml

```diff
@@ -2966,7 +2980,22 @@ components:
       type: object
       properties:
         access_threshold_users:
-          description: 'users who accessed the site on 4 or more days within a month'
+          description: 'Users who accessed the site on 4 or more days within a month'
+          type: integer
+          example: 500
+    user-metrics-profiles-schema:
+      type: object
+      properties:
+        user_count:
+          description: 'The total number of active users'
+          type: integer
+          example: 500
+        users_with_profile_photos:
+          description: 'Number of profiles with a profile photo'
```

Comment:
I think we need to explicitly state whether this is limited to "active" profiles.